### DISCUSSION TITLE
Xiaomi Vacuum Mop 2 Pro (ijai.vacuum.v3) integration?

### DISCUSSION THREAD

**Discussion by fcastelay** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#407] [Q&A]
Thank you very much for your contribution! my query is because I am not being able to integrate my vacuum Xiaomi Vacuum Mop 2 Pro (ijai.vacuum.v3)  to my home assistant to be able to use your contributions!! Do you know if it can be integrated? Has anyone been able? Thank you very much again!!!

**Comment by PiotrMachowski** [comment]
If you can't find any other way [Xiaomi MIoT](https://github.com/al-one/hass-xiaomi-miot) should work
